style.css

```diff
@@ -0,0 +1,56 @@
+section {
+  background-image: url("images/header-shapes mobile@2x.png");
+  min-height: 718px;
+  max-width: 300px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  padding: 5px;
+}
```

Comment:
- [ ] The elements are not properly aligned. Please use proper alignment properties to follow the original design.
- [ ] The background image also is not properly displayed as the original Figma design. 
![image](https://user-images.githubusercontent.com/64894927/121443764-cb6b0d00-c996-11eb-9c70-ac09ccc9f994.png)   
![image](https://user-images.githubusercontent.com/64894927/121443873-010ff600-c997-11eb-9b61-ba283c143b99.png)